Extension/package.json

```diff
@@ -2003,6 +2007,10 @@
                 "type": "boolean",
                 "description": "%c_cpp.debuggers.requireExactSource.description%",
                 "default": true
+              },
+              "detail": {
```

Comment:
Why are we adding `detail` to the launch configuration? I thought this was just a change so the UI has additional information. 

Any specific detail  to tasks should be in tasks.json, not launch.

Reply:
This is for the F5 (Build and Debug Active File) scenario in which the launch.json needs the details (see the screenshots). tasks.json already behaves correctly. Is there a problem with this?

Reply:
Yes. 

This will add `details` into `launch.json` and it will be outdated if the user changes the command in `tasks.json`. If the user wants to know the exact command when it has ran, the user can see what is executed in the terminal window or just go to the `tasks.json` and look at the `command`.
Here is an image of it being populated in the `launch.json`.
![image](https://user-images.githubusercontent.com/3953714/97263208-afd43d00-17df-11eb-97e7-7999b4a3c3c1.png)

Adding descriptions for it in the debugger schema in `package.json` will create an autocompletion for `detail`. However, this is not used for the debug adapter at all and will be sent as part of the launch configuration.
![image](https://user-images.githubusercontent.com/3953714/97263311-f033bb00-17df-11eb-87a4-d31073a772ce.png)

I highly prefer it to not be written to the `launch.json` or show up as an item you can add in `launch.json`. It should only be used to for the MenuItem since I doubt it will be needed once it has been generated.

Reply:
You need to undo all changes in `package.json`. This will still create an autocomplete prompt for `cppvsdbg`.